script.js

```diff
@@ -1,4 +1,119 @@
+var AWAITING_USER_NAME = "awaiting user name";
+var PSP_GAME = "psp game";
+var currentGameMode = AWAITING_USER_NAME;
+var userName = "";
+
+console.log("global");
+
+var winCount = 0;
+var loseCount = 0;
+var drawCount = 0;
+
+var totalCount = function (winCount, loseCount, drawCount) {
+  return Number(winCount) + Number(loseCount) + Number(drawCount);
+};
+console.log("stats");
+
+var winPercentage = function () {
+  return Math.floor((Number(winCount) * 100) / totalCount);
+};
+
+console.log("percent");
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+
+  if ((currentGameMode = AWAITING_USER_NAME)) {
+    userName = input;
+    currentGameMode = PSP_GAME;
+    myOutputValue =
+      "Welcome to the SPS club, " +
+      userName +
+      "!" +
+      "<br><br>" +
+      "Please start the game by choosing either scissors, paper or stone.";
+
+    console.log(main);
+  } else if ((currentGameMode = PSP_GAME)) {
```

Comment:
Remember to use `==` to check for equality! A single `=` is an assignment operator in JavaScript, not equality (unlike what we learnt in mathematics). Assignment is used to do things like creating variables, or re-assigning values stored in variables:
```
var number = 5;
console.log("number here is: " + number); // should print "number here is: 5"
number = 1 + 2;
console.log("number over there is: " + number); // should print "number over there is: 3"
```